yarn.lock

```diff
@@ -3258,6 +3259,11 @@ core-js@^2.5.7:
   version "2.5.7"
   resolved "https://registry.yarnpkg.com/core-js/-/core-js-2.5.7.tgz#f972608ff0cead68b841a16a932d0b183791814e"
 
+core-js@^3.0.0:
+  version "3.6.5"
+  resolved "https://registry.yarnpkg.com/core-js/-/core-js-3.6.5.tgz#7395dc273af37fb2e50e9bd3d9fe841285231d1a"
+  integrity sha512-vZVEEwZoIsI+vPEuoF9Iqf5H7/M3eeQqWlQnYa8FSKKePuYTf5MWnxb5SDAzCa60b3JBRS5g9b+Dq7b1y/RCrA==
+
```

Comment:
Why are there so many different versions of `core-js`? Does `yarn.lock` just keep everything forever?

Reply:
No, it doesn't keep them. Each version is required by something within the whole dependency tree, and this lock file is what version was used exactly for each version.

In this case for Prebid, it it using version `3.6.5` of `core-js`, whilst other libraries are using `2.x.x`.